src/classes/GE_FormRendererService.cls

```diff
@@ -88,8 +114,7 @@ public with sharing class GE_FormRendererService {
     public static Id saveAndProcessSingleGift(DataImport__c diRecord, String widgetData, Boolean updateGift) {
 
         // Check for widget data and convert it to the correct format for processing
-        String additionalObjectJSON = convertWidgetDataToObjectJSON(widgetData);
-        diRecord.Additional_Object_JSON__c = additionalObjectJSON;
+        convertWidgetDataToObjectJSON(diRecord, widgetData);
```

Comment:
It looks like this is only happening in the single gift path, which explains the difficulty of getting the GAUs to save through Batch; it looks like the save paths may be more out of sync between Single and Batch than I had hoped, but one way or another this needs to work for both!